data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,47 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    last_updated_year: 2020
+    source_url: https://hazards.fema.gov/nri/Content/StaticDocuments/DataDownload//NRI_Table_CensusTracts/NRI_Table_CensusTracts.zip
+    extracted_file_name: "NRI_Table_CensusTracts.csv"
+    description: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    null_representation: "None"
+    load_fields:
+      - short_name: "ex_loss"
+        df_field_name: "RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME"
+        long_name: "FEMA Risk Index Expected Annual Loss Score"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_pop_loss"
+        df_field_name: "EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME"
```

Comment:
The tile name for `EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME` that the FE expects for this is `EPLR_PFS` along with a boolean of `EPL_ET`. Should these values also be included in this datasets.yml file?

Reply:
Great question, and the short answer is we'll figure the side effects in another PR. We might change the names -- or not that the tile file uses, but it will be with your input of course.

Reply:
I think the idea here is the short name is part of the renamed column that we use internally, vs the string that describes the column.

Reply:
I think more than `short_name`, these would become column IDs that we can reference in other ETL or CI/CD processes, so even if we don't use them for the tiles, there's a value to adding them.